src/TodoListItem.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+function TodoListItem(props) {
+
```

Comment:
nit: extra spaces. Also sorry for pointing stuff like this out I know it doesn't effect the functionality of the app. It's just something that will get pointed out when developing professionally. I put nit on it because its not going to keep me from approving the pull request just something to be aware of.